### PR TITLE
use nse_relax_factor for massive star

### PR DESCRIPTION
**Pull request by zingale**
also reduce frequency of output and switch to the new Y_e based initial model

## PR checklist

- [ ] test suite needs to be run on this PR
- [ ] this PR will change answers in the test suite to more than roundoff level
- [ ] all newly-added functions have docstrings as per the coding conventions
- [ ] the `CHANGES` file has been updated, if appropriate
- [ ] if appropriate, this change is described in the docs

### CODE REVIEW

**Comment by zingale** [comment]
depends on https://github.com/starkiller-astro/Microphysics/pull/702

**Comment by zingale** [comment]
this runs for ~1700 coarse timesteps with max_level = 2:

Screenshot from 2021-07-04 09-01-09

**Review by maxpkatz — APPROVED** [review]

**Comment by zingale** [comment]
Note: I just updated these to use a more consistent resolution, so feel free to re-review

File: Exec/science/massive_star/inputs_2d.nse [hunk 1]
```diff
@@ -45,7 +45,7 @@ castro.sponge_upper_density = 1.e3
 castro.sponge_lower_density = 1.e2
 castro.sponge_timescale     = 1.e-3
 
-castro.cfl            = 0.7     # cfl number for hyperbolic system
+castro.cfl            = 0.2     # cfl number for hyperbolic system
```
Comment (maxpkatz) [inline Exec/science/massive_star/inputs_2d.nse:48]: Why does CFL need to be so small?

Comment (zingale) [reply Exec/science/massive_star/inputs_2d.nse:48]: cause burning is hard?

**Review by maxpkatz — APPROVED** [review]

**Pull request merged**